_episodes_rmd/02-r-plotting.Rmd

```diff
@@ -947,12 +1005,13 @@ Now that we've learned how impactful effective data visualization can be, and ho
 
 Pair up with another learner and discuss your data and what type of exploratory data analysis you would like to perform.
 
-Questions to conisder:
+Questions to answer:
 
-1. What information do you have in your data set?
-1. Are there specific questions are you interested in investigating with the data you have?
-1. What visuals can you create to explore your data? Draw some out on paper! What will these plots show you?
-1. Is your data in a format that can easily be read into R? How can you improve the formatting of the contents of the file? 
+1. In 1-2 sentences, describe the information covered in your dataset.
+2. How large is your dataset? How many rows? How many columns?
+3. Write down 3 questions you could answer with your data set.
```

Comment:
Should we be clearer about what types of questions we're thinking of? I think we want them to be 3 specific questions (that can be explored through a specific plot)?
 
```suggestion
3. Write down 3 specific questions you could answer with your data set.
```